.gitignore

```diff
@@ -1,3 +1,4 @@
+/.idea
```

Comment:
Recommend adding `.vscode` as well.

Reply:
.vscode is already there on line 28. I'll drop this duplicate of `.idea`